src/controllers/companyController.js

```diff
@@ -0,0 +1,87 @@
+import { companies } from "../models/index.js";
+import notFoundError from "../erros/notFoundError.js";
```

Comment:
Por notação e boas práticas, é preferível que módulos que representam classes iniciem com letra maiúscula. Logo, te sugiro a aplicar isso também com os erros personalizados:

```suggestion
import NotFoundError from "../erros/notFoundError.js";
```

Reply:
My bad! 

Na hora de refatorar os nomes não me atentei ao PascalCase para as classes